users/kuatsure/kuatsure.h

```diff
@@ -34,6 +39,7 @@ void tmux_pane_zoom(void);
 #define LEADER_TIMEOUT 300
 
 #define LAYOUT_preonic_grid_wrapper(...)   LAYOUT_preonic_grid(__VA_ARGS__)
+#define LAYOUT_planck_grid_wrapper(...)   LAYOUT_planck_grid(__VA_ARGS__)
```

Comment:
I think, ideally, these should go in the keymap files for the specific keyboards.